### ISSUE TITLE
Long waiting time

### ISSUE THREAD

**Issue by KickiLagerstedt** [Clinical-Genomics/cgadmin#14]
... between submitting a project and getting the notification that the project is submitted.

**Comment by robinandeer** [comment]
Yes, I'm aware of this. It's the process of opening a new ticket automatically that takes a long time but this isn't something we control.. 

If possible I will at least try to add some indicator while the order is being processed

**Comment by KickiLagerstedt** [comment]
👍
